src/others/helpers/map-utils.ts

```diff
@@ -1,160 +1,51 @@
-import { groupBy } from "lodash";
-import omit from "lodash/omit";
 import { AidRequest, Supply, Location } from "../contexts/api";
-import { DecodedAidRequest, DecodedAidRequestGroupedByLocation } from "./decode-aid-request";
 import type { Feature, Geometry, GeoJsonProperties } from "geojson";
 
-type GroupedByDate = {
-  date: string;
-  aidRequests: Omit<DecodedAidRequest, "date">[]; // TODO: consider replacing with AidRequest
-};
-
-type GroupedByDateWithTotal = {
-  date: string;
-  aidRequests: Omit<DecodedAidRequest, "date">[]; // TODO: consider replacing with AidRequest
+type GroupedByCityId = {
+  city_id: number;
   total: number;
-  totalDescription: string;
-};
-
-type GroupedByLocationAndDateWithTotal = {
-  location: Location;
-  aidRequestsByDateWithTotal: GroupedByDateWithTotal[];
+  aidRequests: AidRequest[];
 };
 
-interface RequestProperties {
+type MapData = {
+  city_id: number;
   amount: number;
-  normalized_amount?: number;
-  date: string;
-  category: string;
   description: string;
-  city: string;
-}
-
-type GroupedByCityId = {
-  city_id: number;
-  total: number;
-  aidRequests: AidRequest[];
 };
 
-// TODO: Delete once map relies on AidRequest data model
-export const adaptToMap = (
-  aidRequestGroupedByCityId: GroupedByCityId,
-  location: Location | undefined,
+export const aggregateCategories = (
+  aidRequestsGroupedByCityId: GroupedByCityId,
   supplyTranslator: (category_id: string) => Supply | undefined
-): DecodedAidRequestGroupedByLocation => {
-  if (!location) throw new Error(`Location: ${location} is not defined`);
-
+): MapData => {
+  const sortedAidRequests = aidRequestsGroupedByCityId.aidRequests.sort((a, b) => b.requested_amount - a.requested_amount);
+  const description = sortedAidRequests.reduce((d, aidRequest) => {
```

Comment:
**issue**

We don't want to have a formatted strings with "\n" here. We want to be able to format this string in any way we want in the component responsible for rendering the data. Not in the mapper.

Reply:
e were doing this before the change. Can this be tackled separately?

Reply:
Yes, but please add a comment `// TODO` in the code, at least for now. It can be handled later depending on if and how the tooltips will be further developed.

Reply:
done